src/api_manager/service_control/proto.cc

```diff
@@ -1163,17 +1163,15 @@ Status Proto::FillReportRequest(const ReportRequestInfo& info,
     }
   }
 
-  return AppendByConsumerOperations(info, request, current_time);
+  return (info.check_response_info.consumer_project_id.empty())
+             ? Status::OK
+             : AppendByConsumerOperations(info, request, current_time);
 }
```

Comment:
Sorry,  this is very hard to read.   I like to see

if () {
   return
}
return

It is an actual function call.